tools/influx_si/macros.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0"?>
+<macros>
+    <token name="@AUTHORS@">
+.. class:: infomark
+
+**Authors** Serguei Sokol (sokol@insa-toulouse.fr)
+
+.. class:: infomark
+
+---------------------------------------------------
+    </token>
+    <token name="@TOOL_VERSION@">7.0.1</token>
+    <token name="@VERSION_SUFFIX@">1</token>
+    <token name="@COMMAND_OPT@"><![CDATA[
+    $noopt
+    $noscale
+    $fullsys
+    $emu
+    $irand
+    $ln
+    $sln
+    $tikhreg
+    $lim
+    $ffguess
+    $nocalc
+    $addnoise
+    $TIMEIT
+    $prof
+    #if $opt.meth:
+      --meth='$opt.meth'
+    #end if
+    #if $opt.irand:
+      --irand
+    #end if
+    #if $opt.sens:
+      --sens='$opt.sens'
+    #end if
+    #if $opt.cupx:
+      --cupx='$opt.cupx'
+    #end if
+    #if $opt.cupn:
+      --cupn='$opt.cupn'
+    #end if
+    #if $opt.cupp:
+      --cupp='$opt.cupp'
+    #end if
+    #if $opt.clownr:
+      --clownr='$opt.clownr'
+    #end if
+    #if $opt.cinout:
+      --cinout='$opt.cinout'
+    #end if
+    #if $opt.clowp:
+      --clowp='$opt.clowp'
+    #end if
+    #if $opt.np:
+      --np='$opt.np'
+    #end if
+    #if $opt.zc:
+      --zc='$opt.zc'
+    #end if
+    #if $opt.fseries:
+      --fseries='$opt.fseries'
+    #end if
+    #if $opt.iseries:
+      --iseries='$opt.iseries'
+    #end if
+    #if $opt.seed:
+      --seed='$opt.seed'
+    #end if
+    #if $opt.excl_outliers:
+      --excl_outliers $opt.excl_outliers
+    #end if
+    #if $opt.tblimit:
+      --tblimit='$opt.tblimit'
+    #end if
+]]>
+    </token>
+    <token name="@MTF_OPT@"><![CDATA[
+        #if $opt.mtf:
+            --mtf '$opt.mtf'
+        #end if
+]]>
+    </token>
+    <xml name="inputs">
+        <inputs>
+            <param type="data" format="zip" name="input_main" multiple="true" label="At least one MTF collection (.netw, .miso, ...)"/>
+            <conditional name="si">
+                <param name="s_i" type="select" label="labeling type" display="radio">
+                    <option value="s">stationary</option>
+                    <option value="i">instationary</option>
+                </param>
+                <when value="s"/>
+                <when value="i">
+                    <param name="time_order" type="select" label="--time_order" display="radio" help="Time order for ODE solving (1 (default), 2 or 1,2). Order 2 is more precise but more time consuming. The value &#x27;1,2&#x27; makes to start solving the ODE with the first order scheme then continues with the order 2.">
+                        <option value="None">None</option>
+                        <option value="1">1</option>
```

Comment:
```suggestion
                        <option value="1" selected="true">1 (default)</option>
```

Reply:
I have to keep 'None' option as this parameter can come also from an input file .opt. In this case, it is not desirable that command line option overwrites by defaults what was possibly given in .opt. I modified the labels to make this situation more clear.